atomic_reactor/plugins/pre_add_yum_repo_by_url.py

```diff
@@ -52,8 +52,20 @@ def run(self):
         if self.repourls:
             for repourl in self.repourls:
                 yumrepo = YumRepo(repourl)
-                yumrepo.fetch()
-                self.log.info("fetched repo from '%s'", yumrepo.repourl)
+                self.log.info("fetching repo from '%s'", yumrepo.repourl)
+                try:
+                    yumrepo.fetch()
+                except Exception as e:
+                    self.log.warning(
```

Comment:
Seems like `self.log.exception` would be useful here, especially because this is an error and not a warning.

Reply:
Ok